src/praisonai-agents/praisonaiagents/memory/memory.py

```
@@ -794,23 +816,24 @@ def store_long_term(
-                return
+        if not adapter_success:
+            try:
+                conn = self._get_ltm_conn()
+                with self._write_lock:  # Serialize write operations
+                    conn.execute(
+                        "INSERT INTO long_mem (id, content, meta, created_at) VALUES (?,?,?,?)",
+                        (ident, text, json.dumps(metadata), created)
+                    )
+                    conn.commit()
+                logger.info(f"Successfully stored in SQLite with ID: {ident}")
+            except Exception as e:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)